### DISCUSSION TITLE
MQTT - HomeAssistant

### DISCUSSION THREAD

**Discussion by Manos1966** [skippermeister/OpenDTU-OnBattery#3] [Q&A]
Hallo @skippermeister 

wie sieht es mit der MQTT Option aus?
- werden alle RS485 Daten per MQTT übermittelt?
- werden sie auch von Home Assistant erkannt, oder muss ich Skripts definieren?

**Comment by skippermeister** [comment]
Es sollten alle Werte per MQTT gesendet werden, von jeder Batterie in Gruppen aufgeteilt. Home Assistant habe ich in meiner Version per Compiler Switch in platform_ override.ini deaktiviert.  Ich bin mir nicht sicher ob ich die Home Assistant Funktion realisiert habe. Das wäre aber einfach zu realisieren. Fragt sich nur ob der Speicher noch ausreicht.

**Reply by skippermeister** [reply]
Ich habe eben mir das Hass Modul für die Batterie Provider angesehen. Ich verstehe noch nicht ganz wie Hass funktioniert, aber es scheint nur rudimentär implementiert zu sein. Es sind nicht alle Werte und Parameter implementiert. Auch sehe ich keine Berücksichtigung mehrerer Pylontech Batterien
